### PR TITLE
[ExpressionLanguage] Add logical xor

### PR DESCRIPTION
**Pull request by Fan2Shrek**
Hello, 

This should fix issue #20280

### CODE REVIEW

**Review by xabbuh — APPROVED** [review]

File: reference/formats/expression_language.rst [hunk 1]
```diff
@@ -489,6 +494,8 @@ The following table summarizes the operators and their associativity from the
 +-----------------------------------------------------------------+---------------+
 | ``or``, ``||``                                                  | left          |
 +-----------------------------------------------------------------+---------------+
+| ``xor``                                                         | left          |
```
Comment (xabbuh) [inline reference/formats/expression_language.rst:495]: rereading the PR I guess that this line needs to be moved up as the table is order by operator precedence

**Review by xabbuh — CHANGES_REQUESTED** [review]

File: reference/formats/expression_language.rst (hunk 1 above)
Comment (Fan2Shrek) [reply reference/formats/expression_language.rst:495]: Does it look good now?

**Review by xabbuh — APPROVED** [review]

File: reference/formats/expression_language.rst (hunk 1 above)
Comment (xabbuh) [reply reference/formats/expression_language.rst:495]: yes, thanks 👍

**Comment by javiereguiluz** [comment]
Pierre, thanks a lot for adding the docs of this new operator ... and congrats on your first Symfony Docs contribution 🎉

**Pull request merged**